CHANGELOG.md

```diff
@@ -14,6 +15,106 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+## v0.5.0
+
+API version: v1alpha2
+
+This release contains a variety of improvements, bug fixes and documentation.
+
+This release includes a large number of breaking changes in preparation for
+an upcoming `beta` release of APIs.
```

Comment:
Maybe a more descriptive intro would help here? A rough idea:

```
This release is all about stability. The changes in this release can largely be divided into the following categories:

- Resources graduating to beta
- New release channels
- New experimental features
- UX improvements for status and kubectl
- Validation improvements
- Internal type cleanup
```

Then maybe the changes below could roughly be structured under subheadings for each of those? I don't know, just an idea. I think the most important thing to highlight is that this release includes resources graduating to beta.

Reply:
Sounds good.

```suggestion
This release is all about stability. The changes in this release can largely be divided into the following categories:

- Resources graduating to beta
- New release channels
- New experimental features
- UX improvements for status and kubectl
- Validation improvements
- Internal type cleanup
```